pkg/cmd/pulumi/new.go

```diff
@@ -572,6 +582,30 @@ func errorIfNotEmptyDirectory(path string) error {
 
 func validateProjectName(ctx context.Context, b backend.Backend,
 	orgName string, projectName string, generateOnly bool, opts display.Options,
+) error {
+	handleExistingProjectName := func(projectName string) error {
+		prompt := "\b\n" + opts.Color.Colorize(colors.SpecWarning+"warning: "+colors.Reset) +
+			fmt.Sprintf("A project with the name `%s` already exists.\n", projectName) +
+			opts.Color.Colorize(colors.SpecPrompt+"Do you want to continue?"+colors.Reset)
+		response := promptUser(prompt, []string{_cont, _edit}, _cont, opts)
+
+		if response != _cont {
+			return fmt.Errorf("already in use")
```

Comment:
Should this not be the same error as non-interactive? "a project with this name already exists"